AUTHORS

```diff
@@ -0,0 +1,8 @@
+Maintainers
```

Comment:
Will this file appear on every repo that we create?

Reply:
yes, but it's a a tempalte, things can be removed/adapted as needed

Reply:
I've replaced by a placeholder/sample.